### ISSUE TITLE
Node 14 - Knex PG can't connect

### ISSUE THREAD

**Issue by nahtnam** [knex/knex#3836]
# Environment

Knex version: 0.20.13
Database + version: pg, 12.1
OS: MacOS Catalina

# Bug

1. Explain what kind of behaviour you are getting and how you think it should do

Migrating and even querying does not work with node 14

2. Error message

```
➜  api git:(master) ✗ nvm use 10
Now using node v10.18.0 (npm v6.13.4)
➜  api git:(master) ✗ knex migrate:latest --env development
Requiring external module ts-node/register
Using environment: development
Already up to date
➜  api git:(master) ✗ nvm use 14
Now using node v14.1.0 (npm v6.14.4)
➜  api git:(master) ✗ knex migrate:latest --env development
Requiring external module ts-node/register
Using environment: development
KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
    at Client_PG.acquireConnection (/PROJECT_ROOT/node_modules/knex/lib/client.js:347:26)
    at async Promise.all (index 1)
```

3. Reduced test code, for example in https://npm.runkit.com/knex or if it needs real
   database connection to MySQL or PostgreSQL, then single file example which initializes
   needed data and demonstrates the problem.

TODO (sorry, haven't had time to do it yet)

**Comment by venables** [comment]
@nahtnam This appears to be [an issue with node-postgres](https://github.com/brianc/node-postgres/issues/2170), not with Knex, and should be fixed in already, just use  **pg v8.0.3** or higher.

**Comment by nahtnam** [comment]
Didn't realize that, thank you!
